### PR TITLE
fix styling

### PR DESCRIPTION
**Pull request by deetz99**
fix modal/table overflowing

### CODE REVIEW

File: web/admin/components/Sbc/Modal/AnnualReportDetails/index.vue [hunk 1]
```diff
@@ -53,7 +53,7 @@ const errorText = computed(() => useErrorMessage(error.value as ApiError, t('pag
 </script>
 <template>
   <UModal v-model="modalModel" :fullscreen="isSmallScreen">
-    <UCard :ui="{ base: 'min-w-fit overflow-hidden', body: { base: isSmallScreen ? 'h-screen space-y-2' : 'max-h-[75vh] space-y-2 overflow-y-auto' }}">
+    <UCard :ui="{ base: isSmallScreen ? 'h-screen overflow-y-auto' : 'min-w-fit overflow-hidden', body: { base: isSmallScreen ? 'space-y-2' : 'max-h-[75vh] space-y-2 overflow-y-auto' }}">
```
Comment (deetz99) [inline web/admin/components/Sbc/Modal/AnnualReportDetails/index.vue:56]: fix not being able to scroll when modal is fullscreen on mobile

File: web/admin/components/Sbc/Modal/AnnualReportDetails/index.vue [hunk 2]
```diff
@@ -135,7 +135,7 @@ const errorText = computed(() => useErrorMessage(error.value as ApiError, t('pag
               v-if="showRawYears.includes(item.header.filingYear)"
               :title="$t('page.admin.table.invite.detailModal.section.rawData.title')"
             >
-              <pre class="mt-1"> {{ item }} </pre>
+              <pre class="mt-1 max-w-[90vw] overflow-x-auto"> {{ item }} </pre>
```
Comment (deetz99) [inline web/admin/components/Sbc/Modal/AnnualReportDetails/index.vue:138]: fix json overflowing screen width on mobile

File: web/admin/components/Sbc/Table/Invitations/index.vue [hunk 3]
```diff
@@ -75,6 +75,7 @@ const errorText = computed(() => useErrorMessage(error.value as ApiError, t('pag
 
   <SbcPageSectionCard
     :heading="$t('labels.searchResults', { count: invitations?.total || 0 })"
+    class="overflow-hidden"
```
Comment (deetz99) [inline web/admin/components/Sbc/Table/Invitations/index.vue:78]: fix table overflowing card on mobile

File: web/admin/pages/admin/index.vue [hunk 4]
```diff
@@ -7,10 +7,6 @@ useHead({
   title: t('page.admin.title')
 })
 
-definePageMeta({
```
Comment (deetz99) [inline web/admin/pages/admin/index.vue:10]: remove unnecessary definePageMeta

**Review by thorwolpert — APPROVED** [review]

**Pull request merged**